src/components/select/select.component.spec.ts

```diff
@@ -229,6 +231,22 @@ describe('Select Component', () => {
         expect(component.value).toBe('One');
     });
 
+    it('should not open dropdown when tabbing past select', (done) => {
+        for (let i = 0; i < 10; i++) {
+            dispatchKeyboardEvent(document, 'keydown', TAB);
+            fixture.detectChanges();
+        }
+
+        setTimeout(() => {
+            expect(getTypeahead()).toBeFalsy();
+            done();
+        }, 1000);
```

Comment:
Just curious, does this need to be `1000`? I understand we may need to delay slightly, but would a smaller timeout work? eg. `1`, or perhaps something even quicker like `Promise.resolve().then(() => ...)` or `requestAnimationFrame`.

While this seems like a small timeout, the more and more tests we add, and every time we delay by a second that can really increase the total time it takes to run the tests on our already slow build process.